runtime/near-test-contracts/build.rs

```diff
@@ -43,7 +43,6 @@ fn build_contract(dir: &str, args: &[&str], output: &str) -> Result<(), Error> {
 
 fn cargo_build_cmd(target_dir: &Path) -> Command {
     let mut res = Command::new("cargo");
-    res.env("RUSTFLAGS", "-C link-arg=-s");
     res.env_remove("CARGO_ENCODED_RUSTFLAGS");
```

Comment:
It seems that we either want to have env_remove("RUSTFLAGS") or get rid of the env_remove("CARGO_ENCODED_RUSTFLAGS") here.  Probably the latter?